CHANGELOG

```diff
@@ -485,7 +485,7 @@ Version 3.3.3 (2015-02-24)
 	Share-it button support (annando)
 	More reliable reshare from Diaspora (annando)
 	Load more images via proxy (annando)
-	util/typo.php uses "php -l" insead of "eval()" to validate code (fabrixxm)
```

Comment:
please revert this change! Historc documents shall not be altered in retrospect, but only with a new entry labeled witht eh new version/date tag.